examples/shop-page/Products.lua

```diff
@@ -0,0 +1,38 @@
+return {
+	{
+		identifier = "Red Visor",
+		image = "https://www.roblox.com/asset-thumbnail/image?assetId=139618072&width=420&height=420&format=png",
+		price = 30,
+		productId = 0,
```

Comment:
should we use real productIds and use those as the stable keys for the list items so we can get rid of the identifier field?

Reply:
Hum I could put random values... Even if we keep the `0` values I could use the list index as a key and get rid of the identifier.

I think at first my plan was to show the name identifier in a label but I dropped that.